src/apps/HeyGPT/HeyGPT/Activation/FileTypeActivationHandler.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Windows.ApplicationModel.Activation;
+using Microsoft.Windows.AppLifecycle;
+using AndGPT.Core.Contracts.Services;
+using AndGPT.Core.Events;
+using AndGPT.Core.Models;
+
+namespace HeyGPT.App.Activation;
+
+public sealed class FileTypeActivationHandler : ActivationHandler<AppActivationArguments>
+{
+    private const string HandlerName = "File";
```

Comment:
This is a new activation handler that will handle any file type related activations.   We have registered the *.heygpt" file type.  Whenever, the user double clicks on that file the HeyGPT application is activated and the information is injected into the running application.   When the activation type is "File" the ActivationService will cause this code to be executed.